docs/versioned_docs/version-2.1/api/hooks.md

```diff
@@ -15,13 +15,10 @@ Check out the [events section](./events.md) for a full list of supported events.
 ```jsx
 import React, { useState } from 'react';
 import { Text, View } from 'react-native';
-import { useTrackPlayerEvents, TrackPlayerEvents, STATE_PLAYING } from 'react-native-track-player';
+import { useTrackPlayerEvents, Event, STATE_PLAYING } from 'react-native-track-player';
```

Comment:
Nice catch!

Could you also:

1. Update this in [the `next` version of the docs as well](https://github.com/doublesymmetry/react-native-track-player/blob/main/docs/docs/api/hooks.md?plain=1#L18)
2. Update the `STATE_PLAYING` to the correct `State.Playing` constant as well (in both places).

Thanks again for this!

State.Playing
--


<br class="Apple-interchange-newline">